### ISSUE TITLE
Edition statements that do not to apply the whole manifestation

### ISSUE THREAD

**Issue by mjmascaro** [rbms-bsc/DCRMR#32]
Policy Statement writers introduced some modifications from DCRM(B) practice for recording edition statements to take advantage of the fact the MARC field 250 is now repeatable has a $3.   These modified instructions have been included in DCRMR. Given the extent of the changes and that it is slightly different approach than taken in RDA proper, feedback from the editorial group at large is requested.

Significant changes from DCRM(B) practice in current draft DCRMR:
[Edition statements on multipart monographs](https://docs.google.com/document/d/1fpj-4fm49I21_VZy9wdAofmHv5ebvXUVgW5YNaVTLwc/edit#bookmark=id.zegixnh6d4qv)
For a multipart monograph, if the edition statement varies or does not appear on all of the volumes. … If the manifestation was issued as such, transcribe all edition statements. Indicate the edition statements corresponding to individual volumes through encoded data and/or a note on edition statement. 
250 __ $3 v. 1 $a The second edition
250 __ $3 v. 2 $a The first edition
Comment: Note on edition statement expressed through MARC subfield 3.

_In DCRM(B), only the edition statement from the first volume was recorded in the edition statement element and variations appearing in subsequent volumes were recorded in a note._

[Two or more works on the preferred source of information with at least one edition statement](https://docs.google.com/document/d/1fpj-4fm49I21_VZy9wdAofmHv5ebvXUVgW5YNaVTLwc/edit#bookmark=id.e083dz7bklro)
 If the preferred source of information bears the titles of two or more individual works contained in the manifestation, and one or more of these works has an edition statement associated with it, transcribe each edition statement that is grammatically linked to another element (e.g. title, statement of responsibility, or any other transcribed element) within that element. Transcribe edition statements not grammatically linked to another element as individual statements. Indicate the element to which each edition statement corresponds in a note on edition statement.

_In  DCRM(B), such editions were all transcribed in the title and SOR area, regardless of grammatical linkage._  

**In both these cases, edition statements related to only part of the manifestation are recorded in the edition statement element.**

The RDA instructions (both in the Pre 3R and Post 3R Toolkit are), which are also included in the current draft DCRMR text:

[Edition statements relating to issues or parts](https://docs.google.com/document/d/1fpj-4fm49I21_VZy9wdAofmHv5ebvXUVgW5YNaVTLwc/edit#bookmark=id.89g8ke1rc68u)

If the manifestation consists of multiple issues or parts, including accompanying material, and there are edition statements relating to the whole as well as to parts, then record only the edition statements relating to the whole manifestation.

Make a note on edition statement relating to issues or parts if considered important.

_In other words, edition statements related to parts are records in note on edition statement, not the edition statement element, even though we have the encoding capability to record multiple edition statements and identify on which subparts they appear_.

**Questions:**

1. Is it permissible in the RDA data model to record edition statements related to issues or parts in the edition statement element since they do not apply to the manifestation as a whole? Or is the RDA guideline a very specific situation that does not conflict with the other proposed DCRMR instructions?
2. If it is not a conflict with RDA, is using repeating edition statements (over notes) preferable over the current DCRM(B) practice?

**Comment by ladylazarus3** [comment]
i think the important piece in the DCRM(B) wording is "ascertain whether the set was issued as such" it *might* be *okay* to use the $$3 with a MARC 250 if indeed we knew they were all **supposed** to be that way for whatever reason, right?

"If cataloging a multipart monograph and the edition statement varies or does not appear on all the volumes, ascertain whether the set was issued as such.  If the manifestation was issued as such, transcribe all edition statements. Indicate the edition statements corresponding to individual volumes by supplying a note on edition statement."

i think the real kicker is how do we illustrate (and explain) the repeating edition statements **without** a note on edition statement due to the nature of the MARC encoding and punctuation.  This is a concern for the text of DCRMR itself as well as a lack of knowledge of how these elements (and the relationships between them) would be expressed/displayed in a future (and at this stage, frankly, magical) RDA/RDF-centric discovery layer.

i think the DCRMR example should end up looking something like this:

Example: `v. 1: The second edition`

Example: `v. 2: The first edition`

**Note on edition statement:**  `Vol. 1 issued as the second edition and vol. 2 issued as the first edition.`

**Comment by elizhobart** [comment]
I agree with kalan. I believe that this does fit with RDA, if the manifestation was issued that way. Since MARC now permits repeating 250s, this would be preferable to notes. kalan's example looks good to me.

**Comment by mjmascaro** [comment]
Thanks, kalan and Elizabeth.  I too like kalan's examples and that would be my preference as well.

I wanted to check-in whether it is okay for me to go ahead to finalize the edition statement using repeating edition statements, or if further discussion is still needed.  (I noticed this is not scheduled for the editorial group meeting tomorrow (July 29th), but since there is already a lot of other topics to discuss, I did not know if this is tabled until the next meeting.)

**Comment by elizhobart** [comment]
Mostly, I didn't schedule this tomorrow for time reasons. However, since this is a practice we're carrying over from the PSs and we haven't seen any dissent, you can go ahead and finalize edition statement with repeating edition statements.
